.changeset/changelog-stylelint.js

```diff
@@ -0,0 +1,98 @@
+// @ts-check
+
+// based on https://github.com/changesets/changesets/blob/main/packages/changelog-github/src/index.ts
+
+const { getInfo, getInfoFromPullRequest } = require('@changesets/get-github-info');
+
+/**
+ * @type {import('@changesets/types').ChangelogFunctions}
+ */
+const changelogFunctions = {
+	async getReleaseLine(changeset, _type, options) {
```

Comment:
Sections like `### Patch Changes` can not be omitted for now.

Reply:
I think we can tolerate it.